soc/st/stm32/stm32wbax/hci_if/ll_sys_if_adapt.c

```diff
@@ -11,11 +11,20 @@
 LOG_MODULE_REGISTER(ll_sys_if_adapt);
 
 #include "ll_sys.h"
+#include "ll_intf_cmn.h"
+#if (USE_TEMPERATURE_BASED_RADIO_CALIBRATION == 1)
```

Comment:
Nit: we're on the Zephyr side so we *should* have visibility on the Kconfig symbols?
```suggestion
#if defined(CONFIG_BT_STM32WBA_USE_TEMPERATURE_BASED_RADIO_CALIBRATION)
```
Ditto for all such usage.